### PR TITLE
Couple of UI nits

### PR DESCRIPTION
**Pull request by pdugan20**
Hello it's me, your friendly neighborhood moonlighting developer. Made a couple of small UI nits across the primary page, take a look when you get the chance.

Screenshot:
Screen Shot 2020-09-10 at 2 04 35 PM

### CODE REVIEW

**Review by maniarathi — APPROVED** [review]
Looks good to me! I'd probably defer to @tihuan 's review though since I'm still a noob at this stuff :)

File: frontend/src/components/ProjectList/components/Dataset/style.ts [hunk 1]
```diff
@@ -15,10 +15,18 @@ export const Wrapper = styled.div`
...
   display: flex;
   align-items: center;
-  padding-left: 10px;
+  line-height: 1.3;
+  padding: 12px 0 12px 10px;
+`;
+
+export const NameChild = styled.div`
+  ${fontStyle};
+  display: block;
```
Comment (tihuan) [inline frontend/src/components/ProjectList/components/Dataset/style.ts:28]: ah `div` is `block` by default, so we can drop this 😄 !

**Review by tihuan — APPROVED** [review]
Whoohooo! Always enjoy your PRs, @pdugan20 🤩 🙏 Thanks so much for doing this! LGTMMM

One tiny nit 😆

File: frontend/src/components/ProjectList/components/Dataset/style.ts (hunk 1 above)
Comment (pdugan20) [reply frontend/src/components/ProjectList/components/Dataset/style.ts:28]: You are correct sir!

**Pull request merged**